### PR TITLE
Docker: silence errors for not defined constants on uninstall

### PR DESCRIPTION
**Pull request by kraftbj**
When running `jetpack docker uninstall`, the script will also attempt to remove all multisite-related constants from wp-config.php. Since these constants are not defined at all on single 
site installs, `wp config delete` will output errors. While we echo a note to ignore the errors, let's try to get rid of them.

## Proposed changes:

* Pipe the output of `wp config delete` to grep to remove strings with "not defined" and ` || true` just to keep a 0 status code (not that this matters at this time)

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

no

## Testing instructions:

* Before patch, `jetpack docker up -d && jetpack docker install`
* `jetpack docker uninstall` -- see the errors.
* Apply patch and rerun `jetpack docker install`
* `jetpack docker uninstall` -- no errors
* `jetpack docker install && jetpack docker multisite-convert` -- reinstall and convert to multisite
* `jetpack docker uninstall` -- confirm success messages are displayed.

### CODE REVIEW

File: tools/docker/bin/uninstall.sh [hunk 1]
```diff
@@ -16,15 +16,14 @@ truncate -s 0 /var/www/html/wp-content/debug.log
...
-echo "It's okay to see errors if these did't exist..."
-wp --allow-root config delete WP_ALLOW_MULTISITE
-wp --allow-root config delete MULTISITE
-wp --allow-root config delete SUBDOMAIN_INSTALL
-wp --allow-root config delete base
-wp --allow-root config delete DOMAIN_CURRENT_SITE
-wp --allow-root config delete PATH_CURRENT_SITE
-wp --allow-root config delete SITE_ID_CURRENT_SITE
-wp --allow-root config delete BLOG_ID_CURRENT_SITE
+wp --allow-root config delete WP_ALLOW_McccULTISITE 2>&1 | grep -v "not defined" || true
```
Comment (anomiex) [inline tools/docker/bin/uninstall.sh]: err?
```suggestion
wp --allow-root config delete WP_ALLOW_MULTISITE 2>&1 | grep -v "not defined" || true
```

**Review by anomiex — COMMENTED** [review]
This looks like it works, one typo inline though.

If you want to get a little fancier, you could do like this instead:
```bash
{
    wp --allow-root config delete WP_ALLOW_MULTISITE
    wp --allow-root config delete MULTISITE
    ... etc
} 2>&1 | grep -v "not defined"
```

File: tools/docker/bin/uninstall.sh (hunk 1 above)
Comment (kraftbj) [reply tools/docker/bin/uninstall.sh]: Ugh. Thanks for catching that. My system started acting up and I guess keyboard input did go through into the wrong window. 

I like your fancy though. I'll look at that before requesting a new review.

File: tools/docker/bin/uninstall.sh [hunk 2]
```diff
@@ -1,29 +1,36 @@
 #!/bin/bash
 
+echo "Uninstalling WordPress..."
+echo "Emptying the WordPress DB..."
 # Empty DB
 wp --allow-root db reset --yes
 
-# Ensure we have single-site htaccess instead of multisite,
-# just like we would have in fresh container.
-cp -f /var/lib/jetpack-config/htaccess /var/www/html/.htaccess
```
Comment (anomiex) [inline tools/docker/bin/uninstall.sh:8]: Did you mean to remove this `cp`?

**Review by anomiex — APPROVED** [review]

**Pull request merged**